starter/cms-payload/README.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Payload with Next.js Starter
+slug: cms-payload
+description: Next.js app with Payload, all in one repo.
+framework: Next.js
+useCase:
+  - Starter
+css: CSS Modules
+deployUrl: https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/starter/cms-payload&project-name=cms-payload&repository-name=cms-payload
+demoUrl: https://cms-payload.vercel.app
+relatedTemplates:
+---
+
+# Next + Payload Serverless Demo
+
+This example shows how to utilize `@payloadcms/next-payload` to deploy Payload serverlessly, in the same repo alongside of a NextJS app.
+
+## Demo
+
+https://cms-payload.vercel.app
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/starter/cms-payload&project-name=cms-payload&repository-name=cms-payload)
```

Comment:
Does this work without providing env vars?

Reply:
Only `MONGODB_URI` and `PAYLOAD_SECRET` are required - I updated the `.env.example` file to clarify.